### PR TITLE
Copy minor bits from yapapi

### PR DESCRIPTION
**Pull request by lucekdudek**

### CODE REVIEW

File: golem_core/golem_node.py [hunk 1]
```diff
@@ -21,6 +21,10 @@
 from .low.payment_event_collector import DebitNoteEventCollector, InvoiceEventCollector
 
 
+DEFAULT_DRIVER: str = os.getenv("YAGNA_PAYMENT_DRIVER", "erc20").lower()
+DEFAULT_NETWORK: str = os.getenv("YAGNA_PAYMENT_NETWORK", "rinkeby").lower()
+DEFAULT_SUBNET: Optional[str] = os.getenv("YAGNA_SUBNET", "public")
```
Comment (approxit) [inline golem_core/golem_node.py:26]: Looks like default values can be changed? Not good idea. Let's investigate, where what values are used, and rework that logic. Optionally introduce some function to resolve that values, using under the hood env lookup with default value fallback.

File: golem_core/low/exceptions.py [hunk 2]
```diff
@@ -43,7 +41,11 @@ def __init__(self, network: str, driver: str):
 
         #   NOTE: we don't really care about this sort of compatibility, but this is
         #         a message developers are used to so maybe it's worth reusing
-        msg = str(NoPaymentAccountError(driver, network))
+        (
+            f"No payment account available for driver `{driver}`"
+            f" and network `{network}`"
+        )
```
Comment (approxit) [inline golem_core/low/exceptions.py:47]: Duplicated?

File: golem_core/low/exceptions.py [hunk 3]
```diff
@@ -43,7 +41,11 @@ def __init__(self, network: str, driver: str):
...
+        msg = f"No payment account available for driver `{driver}` and network `{network}`"
```
Comment (approxit) [inline golem_core/low/exceptions.py:48]: Is that variable really needed?

**Review by approxit — CHANGES_REQUESTED** [review]

File: golem_core/low/exceptions.py (hunk 3 above)
Comment (lucekdudek) [reply golem_core/low/exceptions.py:48]: It is like this in all custom exception here

File: golem_core/golem_node.py (hunk 1 above)
Comment (lucekdudek) [reply golem_core/golem_node.py]: Rename to 
```python
PAYMENT_DRIVER: str = os.getenv("YAGNA_PAYMENT_DRIVER", "erc20").lower()
PAYMENT_NETWORK: str = os.getenv("YAGNA_PAYMENT_NETWORK", "rinkeby").lower()
SUBNET: Optional[str] = os.getenv("YAGNA_SUBNET", "public")
```

**Pull request merged**